skietbaan_fe/src/App.js

```diff
@@ -1,46 +1,23 @@
-
-
 import React, { Component } from 'react'
 import { connect } from 'react-redux';
 import './App.scss'
 import { getUser } from './api/github'
 import {BrowserRouter, Route, Switch, Redirect} from 'react-router-dom'
-import Home from "./components/Home"
+import landing from "./components/landing"
```

Comment:
Inconsistent capitalised naming of components please capitalise this and the name if the component if it is not capitalised

Reply:
And also please either add the semi-colons at the end of each import OR dont add them. It looks very ugly like that although you wont have any compiling errors.

Reply:
You can also set up ESLint in your project if you want cleaner code

Reply:
I have changed it